docs/high_availability_installation_guide.md

```diff
@@ -190,15 +190,13 @@ In ```Harbor.cfg```
 Change the value to your Harbor DB's
 
 ```
+#DB configuration Section
```

Comment:
suggestion: we can using ### ... ### to start a section , instead of a single #.